submissions/alliEse/html-forms/contact.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Contact Us - Brogues by Wingtip Brogues</title>
+</head>
+<body>
+<header>
+        <h1>Brogues by Wingtip Brogues</h1>
+</header>
+    <nav>
+        <ul>
+                <li><a href="#"> Home</a></li>
```

Comment:
Please remove the href attribute and its value from all a-tags in the nav. The linking will be done in subsequent tasks.